tests/css_less/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2690,9 +2690,7 @@ div {
 header {
   background-color: blue;
 
-  .desktop-and-old-ie({
-    background-color: red;
-  });
+  .desktop-and-old-ie({background-color: red;});
```

Comment:
We use `postcss-selector-parser` and it is can break code. We should use `postcss-value-parser` for this. I can do this in other PR.